scripts/sign_image_dev.sh

```diff
@@ -0,0 +1,14 @@
+sign_image_dev()
```

Comment:
>sign_image_dev [](http://example.com/codeflow?start=0&length=14)

Better to make this script executable, no need to expose the function to outside.